types/index.d.ts

```diff
@@ -873,7 +873,7 @@ declare namespace jest {
 
   // noinspection JSUnusedGlobalSymbols
   // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  interface Expect extends CustomMatchers {}
+  interface Expect extends CustomMatchers<any> {}
```

Comment:
Used `any` as of this error: https://github.com/jest-community/jest-extended/issues/550#issuecomment-1375482879